### DISCUSSION TITLE
v4 planned release date?

### DISCUSSION THREAD

**Discussion by dario-oppl** [fluentmigrator/fluentmigrator#1647] [Q&A]
Hi team, I love this library been using it for 7 years now.
I would like to know plans to release v4, or where I can get a beta or something?
I'm really willing to upgrade and test.
Is there a place where nuget pre-release packages are pushed?

**Comment by vmandic** [comment]
+1 any info @jzabroski?

edit: I see v4 is skipped for v5: https://github.com/fluentmigrator/fluentmigrator/milestone/28

So, v5, when, aaaany whereabouts?
